core/kazoo_data/src/kz_data.hrl

```diff
@@ -68,6 +68,7 @@
 
 -type view_option() :: 'conflicts' |
                        'descending' |
+                       'first_when_multiple' |
```

Comment:
That's a lie. `first_when_multiple` is already added to the spec of the only function that uses it. Revert this